ta/arch/arm32/fix_ta_binary

```diff
@@ -27,7 +27,7 @@
 use strict;
 use warnings;
 use diagnostics;
-use Env qw($READELF);
+use Env qw($READELF $CROSS_COMPILE_ta);
```

Comment:
Good question, we have `user_ta` in `ta.mk`, but we have `ta` in `ta_dev_kit.mk`. @jenswi-linaro  what was your initial idea here?